src/components/Firestore/Requests/RequestDetails/Actions.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import './Actions.scss';
+
+import { Button } from '@rmwc/button';
+import React from 'react';
+
+import RequestActionsCollapsedMenu from './ActionsCollapsedMenu';
+
+export interface RequestAction {
+  label: string;
+  action: () => void;
+}
+export const REQUEST_ACTIONS: RequestAction[] = [
+  {
+    label: 'Retrigger request',
+    action: () => {},
+  },
+];
+
+export const Action: React.FC<RequestAction> = ({ label, action }) => (
+  <Button
+    outlined
+    onClick={action}
+    className="Firestore-Request-Details-Action"
+    role="request-details-action-button"
+  >
+    {label}
+  </Button>
+);
+
+const RequestActions: React.FC = () => (
```

Comment:
While this is nice and functional, we won't be able to implement this for real in quite a while. What's your plan for hiding / disabling it for now? We certainly don't want to ship with a button that doesn't do anything yet.

Reply:
The Action buttons will be included in this PR, but all components whose functionality is not yet ready or are simply not ready to be shipped, will be disabled in a later PR.